src/App.vue

```diff
@@ -1,7 +1,71 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+  import { ref } from 'vue';
+  import SnackfyVue from './components/SnackfyVue.vue';
+
+  const SnackfyVueRef = ref<typeof SnackfyVue>(SnackfyVue);
```

Comment:
I'll have to look for a different approach in the future, but that's how I've managed to keep the logic contained without using stores.